baseplate/integration/pyramid.py

```diff
@@ -107,13 +127,23 @@ class BaseplateConfigurator(object):
 
     """
 
-    # TODO: remove the default on trust_trace_headers once apps are updated
     def __init__(self, baseplate, trust_trace_headers=False,
-                 edge_context_factory=None):
+                 edge_context_factory=None, header_trust_handler=None):
         self.baseplate = baseplate
         self.trust_trace_headers = trust_trace_headers
+        if self.trust_trace_headers:
```

Comment:
You might need to make the default value `None` and have this check be `if trust_trace_headers is not None` and set `self.trust_trace_headers = bool(trust_trace_headers)`